delivery/manifest/manifest.yaml

```diff
@@ -7,21 +7,21 @@ metadata:
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: helloservice
+  name: podtato-main
   namespace: demospace  
 spec:
   selector:
     matchLabels:
-      app: helloservice
+      app: podtato-main
   template:
     metadata:
       labels:
-        app: helloservice
+        app: podtato-main
     spec:
       terminationGracePeriodSeconds: 5
       containers:
       - name: server
-        image: ghcr.io/podtato-head/podtatoserver:v0.1.1
+        image: gdsoumya/podtato-main:v0.1.0
```

Comment:
I think the image should stay in the podtato-head repo

Reply:
yup missed that, fixed